### PR TITLE
Definition for mTLS (Mutual Transport Layer Security).md

### PR DESCRIPTION
**Pull request by Alt-Shivam**
Issue Details- mTLS definition #252

### CODE REVIEW

**Comment by JasonMorgan** [comment]
@Alt-Shivam I sent you some feedback in slack. Beyond that I'd also like to see the how it helps section be simpler to understand for folks not already familiar with TLS or microservices.

**Comment by Alt-Shivam** [comment]
Hey @JasonMorgan  ,I have made certain changes to the file,
and I think it's more beginner-friendly now,
Have a look

**Comment by CathPag** [comment]
@Alt-Shivam, thanks for your contribution. Here's some suggetions:

The wording of the first section was still too close to the original source. Below is a suggested update that changes that AND uses more simple words. I also eliminated the SSL reference as it adds more complexity without explaining what it is. 

## What it is

Mutual TLS (mTLS) is a technique used to authenticate and encrypt messages sent between two [services](https://glossary.cncf.io/service/). Mutual TLS is the standard Transport Layer Security (TLS) protocol but, instead of validating the identity of just one connection, both sides are validated.  

****

For the second section, I added a wifi reference which helps illustrate what we are talking about. Everything about the goal of TLS on the internet belongs in a TLS definition, not here (we need one for the glossary as well). Instead we should add here why and when two-way authentication is necessary. 

## Problem it addresses

[Microservices](https://glossary.cncf.io/microservices/) communicate over a network and, just like your wifi network, communication in transit over that network can be hacked. mTLS ensures that no unauthorized party can send potentially malicious [API](https://glossary.cncf.io/application_programming_interface/) requests. [WHY IS TWO-WAY AUTHENTICATION NECESSARY?]

****
In the last section, I tweaked the first sentence which was still too close to the original source and fixed capitalization in the last sentence.

## How it helps

mTLS ensures that traffic is secure and trusted in both directions between a client and server, providing an additional layer of security for users who log in to a network or applications. It also verifies connections with client devices that do not follow a login process, such as Internet of Things (IoT) devices. Attacks like on-path attacks, spoofing attacks, credential stuffing, brute force attacks, malicious API requests, etc. can be prevented by mTLS.

**Comment by Alt-Shivam** [comment]
Thanks, @CathPag for the help,
one thing I want to ask,
Should I add [WHY IS TWO-WAY AUTHENTICATION NECESSARY?] to this,
or an external link is preferable.

**Comment by Alt-Shivam** [comment]
Hey @CathPag, Thanks a lot for the help.
I have committed the changes,
Have a look now.

**Review by CathPag — APPROVED** [review]
Thank you, looks great, @Alt-Shivam! Let's see what @JasonMorgan says. If he's on board, we'll, hopefully, get this merged this week :)

File: content/en/mTLS (Mutual Transport Layer Security).md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: mTLS (Mutual Transport Layer Security)
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+Mutual TLS (mTLS) is a technique used to authenticate and encrypt messages sent between two [services](https://glossary.cncf.io/service/). Mutual TLS is the standard Transport Layer Security (TLS) protocol but, instead of validating the identity of just one connection, both sides are validated.
+
+## Problem it addresses
+[Microservices](https://glossary.cncf.io/microservices/) communicate over a network and, just like your wifi network, communication in transit over that network can be hacked. mTLS ensures that no unauthorized party can send potentially malicious [API](https://glossary.cncf.io/application_programming_interface/) requests.
```
Comment (JasonMorgan) [inline content/en/mTLS (Mutual Transport Layer Security).md:11]: Can we change "mTLS ensures that no unauthorized party can send potentially malicious [API](https://glossary.cncf.io/application_programming_interface/) requests." to "mTLS ensures that no unauthorized party can listen in on or impersonate legitimate requests."

File: content/en/mTLS (Mutual Transport Layer Security).md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## How it helps
+mTLS ensures that traffic is secure and trusted in both directions between a client and server, providing an additional layer of security for users who log in to a network or applications. It also verifies connections with client devices that do not follow a login process, such as Internet of Things (IoT) devices. Attacks like on-path attacks, spoofing attacks, credential stuffing, brute force attacks, malicious API requests, etc. can be prevented by mTLS.
```
Comment (JasonMorgan) [inline content/en/mTLS (Mutual Transport Layer Security).md:14]: Let's drop "malicious API requests" as mTLS will only handle ensuring everyone is who they say they are, it won't help if one party is malicious.

**Review by JasonMorgan — CHANGES_REQUESTED** [review]
Just about there. Great work. Can you also update the status to "Completed"?

File: content/en/mTLS (Mutual Transport Layer Security).md (hunk 1 above)
Comment (Alt-Shivam) [reply content/en/mTLS (Mutual Transport Layer Security).md:11]: Sure :)

**Pull request closed without merging**